lib/components/narrative/default/itinerary-summary.js

```diff
@@ -1,6 +1,19 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
+function getModeColor (mode) {
+  switch (mode) {
+    case 'BICYCLE': return '#E0C3E2'
+    case 'BUS': return '#CAC3DF'
+    case 'CAR': return '#E4CCCC'
+    // case 'PARK': return '#E4CCCC'
+    case 'RAIL': return '#BDDAC0'
+    case 'WALK': return '#DFC486'
+    default:
+      return 'grey'
+  }
```

Comment:
It'd probably be better to use a hashmap here so that each case statement doesn't need to be analyzed.